packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerOptions.kt

```diff
@@ -22,6 +22,10 @@ internal class ImagePickerOptions : Record, Serializable {
   @FloatRange(from = 0.0, to = 1.0)
   var quality: Double = 0.2
 
+  @Field
+  @IntRange(from = 0)
+  var selectionLimit: Int = 0
```

Comment:
If `0` indicates no limit, could you please add a comment here or make it like on iOS where it's set to `UNLIMITED_SELECTION` constant?https://github.com/expo/expo/blob/887aceac3db640a2900961ccff84eebe63e718c3/packages/expo-image-picker/ios/ImagePickerOptions.swift#L54